### PR TITLE
feat: changed to typeorm

### PR DESCRIPTION
**Pull request by llong2195**
Changed to use TypeOrm and Mysql
- Create migration: yarn typeorm:create apps/api/src/app/database/migrations/createTableUser
- Run migration: typeorm:up

### CODE REVIEW

File: apps/api/src/app/database/database.module.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+
+@Module({
+  imports: [
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule],
+      useFactory: async (
+        configService: ConfigService
+      ): Promise<TypeOrmModuleOptions> => ({
```
Comment (tuanbm) [inline apps/api/src/app/database/database.module.ts]: Change to support mysql replication configuration

**Review by tuanbm — CHANGES_REQUESTED** [review]

**Comment by tuanbm** [comment]
```
Create migration: yarn typeorm:create apps/api/src/app/database/migrations/createTableUser
Run migration: typeorm:up
```
Change the create command to remove (not require) the absolute path, just the name.

**Comment by llong2195** [comment]
> ```
> Create migration: yarn typeorm:create apps/api/src/app/database/migrations/createTableUser
> Run migration: typeorm:up
> ```
> 
> Change the create command to remove (not require) the absolute path, just the name.

fix https://github.com/OnionStudios/example-nx-app/pull/5

**Comment by llong2195** [comment]
> > ```
> > Create migration: yarn typeorm:create apps/api/src/app/database/migrations/createTableUser
> > Run migration: typeorm:up
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Change the create command to remove (not require) the absolute path, just the name.
> 
> add #5

**Pull request merged**